core/gui/src/app/workspace/service/report-generation/report-generation.service.ts

```diff
@@ -260,6 +281,9 @@ export class ReportGenerationService {
   <html>
     <head>
       <title>Operator Results</title>
+      <!-- Link to JSONEditor CSS file-->
+      <link href="https://cdn.jsdelivr.net/npm/jsoneditor@latest/dist/jsoneditor.min.css" rel="stylesheet" type="text/css" />
+      <script src="https://cdn.jsdelivr.net/npm/jsoneditor@latest/dist/jsoneditor.min.js"></script>
```

Comment:
Ditto

Reply:
Using a CDN to include CSS files is a common practice. The URL is from jsdelivr.
 And Jsdeliver is very common used, and generally reliable.